modules/ip/cli/address.c

```
@@ -49,17 +48,20 @@ static int addr_flush(struct gr_api_client *c, uint16_t iface_id) {
 	return gr_api_client_send_recv(c, GR_IP4_ADDR_FLUSH, sizeof(req), &req, NULL);
 }
 
-static int addr_list(struct gr_api_client *c, uint16_t iface_id, struct libscols_table *table) {
+static int addr_list(struct gr_api_client *c, uint16_t iface_id, struct gr_table *table) {
 	struct gr_ip4_addr_list_req req = {.vrf_id = GR_VRF_ID_UNDEF, .iface_id = iface_id};
 	const struct gr_ip4_ifaddr *addr;
 	int ret;
 
 	gr_api_client_stream_foreach (addr, ret, c, GR_IP4_ADDR_LIST, sizeof(req), &req) {
 		if (iface_id != GR_IFACE_ID_UNDEF && addr->iface_id != iface_id)
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)